wiki/Client/Beatmap_editor/Song_Setup/fr.md

```diff
@@ -250,7 +252,7 @@ En déplaçant le slider vers la droite, vous espacerez la "proximité" des obje
 
 *Remarque : Les cercles empilés dans osu! se déplaceront vers le point où l'empilement se produit.*
 
-Les objets sont considérés comme appartenant à la même pile s'ils se trouvent à la même coordonnée et pas plus loin dans le temps que `[(Fenêtre d'approche) * (Limite de pile) / 10]`, où `Fenêtre d'approche` est la durée pendant laquelle les objets restent à l'écran (voir [Approach Rate](#approach-rate).
+Les objets sont considérés comme appartenant à la même pile s'ils se trouvent à la même coordonnée et pas plus loin dans le temps que `[(Fenêtre d'approche) * (Limite de pile) / 10]`, où `Fenêtre d'approche` est la durée pendant laquelle les objets restent à l'écran (voir [La section sur l'approach rate](#approach-rate).
```

Comment:
here it's "approach rate", while elsewhere you've changed it to "Approach rate". In english, these have slowly been transitioning towards using all lowercase when not written as an acronym, but should they have the first letter in capital in french?

Reply:
It's not intentional, no, I thought I had done the opposite (put them in lower case), I rectify that